### PR TITLE
refactor: increase nb of children, for delete, add zip public endpoints

### LINKED ISSUE #167: Increase allowed number of children in an item

### PR DESCRIPTION
**Pull request by pyphilia**
This PR updates dependencies, increase the limit for deletion and creation, register the public endpoint for export/import zip.

close #167

### CODE REVIEW

File: .devcontainer/localstack/init.sh [hunk 1]
```diff
@@ -17,6 +17,19 @@ corsConfig='{
...
+            ],
+            "AllowedMethods": [
+                "HEAD",
+                "GET",
+            ],
+            "AllowedOrigins": [
+                "*"
+            ],
+            "ExposeHeaders": []
         }
```
Comment (spaenleh) [inline .devcontainer/localstack/init.sh:33]: What is the difference between this and the above ? Out of curiosity.

File: test/membership.test.ts [hunk 2]
```diff
@@ -833,39 +833,9 @@ describe('Membership routes tests', () => {
     });
     it(`Cannot delete more than ${MAX_ITEM_MEMBERSHIPS_FOR_DELETE} memberships`, async () => {
       const item = getDummyItem();
-      const memberships = [
+      const memberships = Array.from({ length: MAX_ITEM_MEMBERSHIPS_FOR_DELETE + 1 }, () =>
```
Comment (spaenleh) [inline test/membership.test.ts:836]: Yay loops !

**Review by spaenleh — APPROVED** [review]
Looks good ! Just a comment to understand 😄

File: .devcontainer/localstack/init.sh (hunk 1 above)
Comment (pyphilia) [reply .devcontainer/localstack/init.sh:33]: The rule above allow only origins `null` to manipulate the files (in our case the backend). 
The second rule allow any origin to get the files, important for apps to fetch files.
Note: you can only access files given a signed url, given by the backend.

Comment (spaenleh) [reply .devcontainer/localstack/init.sh:33]: OK ! 🔥

**Pull request merged**